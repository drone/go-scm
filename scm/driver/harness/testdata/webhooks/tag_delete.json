{
  "trigger": "tag_deleted",
  "repo": {
    "id": 16,
    "path": "kmpySmUISimoRrJL6NL73w/harness-core",
    "identifier": "harness-core",
    "default_branch": "develop",
    "git_url": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "uid": "harness-core"
  },
  "principal": {
    "id": 2,
    "uid": "lv0euRhKRCyiXWzS7pOg6g",
    "display_name": "Admin",
    "email": "admin@harness.io",
    "type": "user",
    "created": 1701091219051,
    "updated": 1701091219051
  },
  "ref": {
    "name": "refs/tags/asd",
    "repo": {
      "id": 16,
      "path": "kmpySmUISimoRrJL6NL73w/harness-core",
      "identifier": "harness-core",
      "default_branch": "develop",
      "git_url": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
      "uid": "harness-core"
    }
  },
  "sha": "0000000000000000000000000000000000000000",
  "old_sha": "3e4da2d65c3631c3e84b52fabe714c978aff540b",
  "forced": false
}