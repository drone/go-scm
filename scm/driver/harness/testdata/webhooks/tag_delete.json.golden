{
  "Ref": "refs/tags/asd",
  "Before": "3e4da2d65c3631c3e84b52fabe714c978aff540b",
  "After": "0000000000000000000000000000000000000000",
  "Repo": {
    "ID": "16",
    "Namespace": "",
    "Name": "harness-core",
    "Perm": null,
    "Branch": "develop",
    "Private": false,
    "Clone": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "CloneSSH": "",
    "Link": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "Commit": {
    "Sha": "",
    "Message": "",
    "Author": {
      "Name": "",
      "Email": "",
      "Date": "0001-01-01T00:00:00Z",
      "Login": "",
      "Avatar": ""
    },
    "Committer": {
      "Name": "",
      "Email": "",
      "Date": "0001-01-01T00:00:00Z",
      "Login": "",
      "Avatar": ""
    },
    "Link": ""
  },
  "Sender": {
    "ID": "lv0euRhKRCyiXWzS7pOg6g",
    "Login": "lv0euRhKRCyiXWzS7pOg6g",
    "Name": "Admin",
    "Email": "admin@harness.io",
    "Avatar": "",
    "Created": "2023-11-27T05:20:19.051-08:00",
    "Updated": "2023-11-27T05:20:19.051-08:00"
  }
}